notebooks/deepmreye_example_usage.ipynb

```diff
@@ -9606,10 +9606,13 @@
     "\n",
     "os.environ[\n",
     "    \"CUDA_VISIBLE_DEVICES\"] = \"0\"  # Change to os.environ[\"CUDA_VISIBLE_DEVICES\"] = \"\" if you dont have access to a GPU\n",
+    "multiprocessing = True # Set to false if running on Apple Silicon (M-series chips)\n",
```

Comment:
Maybe just add a line below:
if is_mac and is_arm:
multiprocessing = False

Then the notebook tests should pass without problems